builder/hyperv/common/driver.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
```

Comment:
@mwhooker is the problem that it is Apache license or is it because its copyright microsoft.

Would like to point out that Azure builder is also contains copyright Microsoft, but its license is MIT,